vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -65,24 +64,77 @@ ent ecr deploy --repo=docker://registry.hub.docker.com/entando/entando-hub-conte
+  7. Note that an admin user needs to belong to an organization as well, especially for private catalogs that require an API key. 
+
+### Create New Entries/Bundle Groups
+Click the `Add +` button at the top of the Hub UI home page to create a new bundle group. In the pop-up window, enter the details for the entry.
+
+![hub-add.png](./hub-images/hub-add.png)
+
+
+1. Upload a file of the thumbnail for the bundle group. 
+2. Add one or more bundles for the entry using the `Add +` button next to the `Add Bundle URI` field.  
+3. Check `Display Contact Us button` and enter the URL under `Contact URL` to gather more information from the viewer/visitor and manage access to the entry. Typically, the contact URL points to a web form on the owner's web site with a request for access to the entry.
+
+### Create a Private Catalog
+A private catalog can be configured in the HUB UI when creating a new organization. There can be many organizations in a single realm of a Hub instance, but each organization can only have one private catalog. Only the Hub admin can create an organization and provision it with a private catalog. 
+1. Go to Organization Management from the top menu
+2. Click `Add Organization +`, enter the relevant information in the pop-up window, and click `Save`
+3. The new organization will appear in the current list. Click on the kebab menu to the right of the organization and select `Create Private Catalog`. 
+A key icon will appear next to the private catalog. To go directly to this catalog, there is a link under the same kebab menu. 
+
+### Generate an API Key
+Private catalogs have the option to utilize an API key instead of user ID specifications for access. The API key can be made mandatory for adding the catalog as a registry in the App Builder. 
+1. Make sure the user, including an admin, had been assigned to the organization with the private catalog.
+2. From the Hub UI homepage, click on the gear icon right of the `Add +` button and select `API Key Management`.
+3. Click `Generate API Key`, enter a name, and confirm with the blue generate button. Save the key for future reference.
+
+### Add a Catalog as a Registry in your App Builder
+Any enterprise Hub instance can be accessed from the Entando App Builder of another Entando Application. Configure the App Builder to access the desired catalog via the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the address for the Entando Application. 
 
-### Concepts
+1. Go to the Hub from the left navigation bar in the App Builder and click `Select Registry` 
+2. Choose `New Registry` from the drop-down menu
+3. Enter a name and URL for the catalog.
 
+     * The address for private catalog follows the same pattern as noted above, but with an added ID number. To find the ID number  in this format: `BASEURL/entando-de-app/en/entando_hub.page#/catalog/YOUR-CATALOGID#/`.  
```

Comment:
How does a user find the ID number in the Hub UI? Seems a bit of guidance may be needed.